packages/replayio/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.4.4
+
+### Patch Changes
+
+- [#572](https://github.com/replayio/replay-cli/pull/572) [`db3f422`](https://github.com/replayio/replay-cli/commit/db3f422416d9385db543e9d4876752a159adfa79) Thanks [@bvaughn](https://github.com/bvaughn)! - Add "whoami" command to print information about the current user and API key
```

Comment:
This should have probably been tagged as a "minor" rather than a "patch"

Reply:
`.changeset` directory is the source of truth. So you can just edit (on `main`) the related changeset file.